### PR TITLE
misc: Allow toggling of memory mapping on PC

### PR DESCRIPTION
**Pull request by Lupiax**

### CODE REVIEW

**Comment by Zangetsu38** [comment]
need squash all, and no need pass config inside it normally, you can using featur for that

**Comment by Zangetsu38** [comment]
Close it, no really needed on pc

**Comment by mrcmunir** [comment]
@Zangetsu38  it's needed for arm devices under PC system  will cause problem in some socs with memory mapping ON   and probably old GPU structure in PC too .
On arm devices due fragmentation drivers you may find that memory mapping via table page works better but introduces new graphical bugs.
Memory mappng via external_memory is quite driver dependent and won't always work depending on the hardware it's a lottery and complete broken if extension detected comming  unstable vita3k compared without it.

File: vita3k/renderer/src/vulkan/gxm_to_vulkan.cpp [hunk 1]
```diff
@@ -480,6 +480,7 @@ namespace texture {
 
 static const vk::ComponentMapping translate_swizzle1(SceGxmTextureSwizzle1Mode mode) {
     switch (mode) {
+    /**/
```
Comment (Zangetsu38) [inline vita3k/renderer/src/vulkan/gxm_to_vulkan.cpp:483]: i juste see here but ???

**Pull request closed without merging**